addon/index.js

```diff
@@ -47,7 +46,7 @@ export default Ember.EventDispatcher.extend({
     //assert('You cannot make a new Ember.Application using a root element that is a descendent of an existing Ember.Application', !rootElement.closest(ROOT_ELEMENT_SELECTOR).length);
     //assert('You cannot make a new Ember.Application using a root element that is an ancestor of an existing Ember.Application', !rootElement.find(ROOT_ELEMENT_SELECTOR).length);
 
-    rootElement.className += ' ' + ROOT_ELEMENT_SELECTOR;
+    rootElement.classList.add(ROOT_ELEMENT_CLASS);
```

Comment:
Thanks for spotting this! I believe that we should probably use `className` still though for cross browser support. Thoughts?